src/mam4xx/lin_strat_chem.hpp

```diff
@@ -273,9 +273,9 @@ Real lin_strat_sfcsink_kk(const Real delta_t, const Real pdel, // in
       (one - haero::exp(-delta_t / o3_tau));     // !compute time scale factor
                                                  //
   const Real mass = pdel * rgrav;                //   air mass in kg/m2
-  const Real o3l_old = o3l_vmr;                  // vmr
+  const Real o3l_old = o3l_vmr_in;                  // vmr
   const Real do3 = (o3_sfc - o3l_old) * efactor; // vmr
-  o3l_vmr = o3l_old + do3;
+  const Real o3l_vmr = o3l_old + do3;
```

Comment:
With this change it does not appear that  `o3l_vmr_in` is an input/output value but input only. Maybe now it should be changed to `const Real.`

Reply:
Thanks. I addressed this comment.